src/savegame_json.cpp

```diff
@@ -1529,21 +1529,62 @@ void item::io( Archive& archive )
 
     // Migrate legacy toolmod flags
     if( !is_toolmod() ) {
-        if( has_flag( "ATOMIC_AMMO" ) ) {
```

Comment:
This section here is superior to my approach as we don't end up with a `TOOLMOD_MIGRATION` flag applied to every tool